ada/language/ast.py

```diff
@@ -407,7 +407,7 @@ def logic_val(from_node=T.AdaNode.entity, lvar=LogicVar,
         success = Var(If(
```

Comment:
English nit about your commit message: You'd say "Rework is_call completely" rt. "Rework completely is_call". Nobody cares though. Just sayin'

Reply:
I think the best one would be "Completely rework is_call" :P

Reply:
is_call, completely. Rework!